src/Kopernicus/Components/ModularScatter/ModularScatter.cs

```diff
@@ -247,7 +247,18 @@ private void Update()
 
             for (Int32 i = 0; i < scatterObjects.Count; i++)
             {
-                if (scatterObjects[i]) continue;
+                if (scatterObjects[i])
+                    if (scatterObjects[i])
```

Comment:
This is only a (botched) revert of commit https://github.com/Kopernicus/Kopernicus/commit/486ca8b846065af946129f02ddf9b445cb33fd4b. 

Like I said in the issue, the important commit to get rid of is https://github.com/Kopernicus/Kopernicus/commit/e5883b23e44db433bd4421db178ec757276834e2. That contains the actual breaking change. The other ones I linked are just further cleanup from Sigma.

So this revery only changes
```csharp
if (scatterObjects[i])
{
    continue;
}
```
to
```csharp
if (scatterObjects[i])
{
    if (scatterObjects[i])
    {
         continue;
    }
}
```